test/unit/content-src/lib/selectLayoutRender.test.js

```diff
@@ -36,21 +37,11 @@ describe("selectLayoutRender", () => {
     assert.deepEqual(result[0].components[0], {type: "foo", feed: {url: "foo.com"}, data: {recommendations: ["foo", "bar"]}});
   });
 
-  it("should return layout property without data if feed isn't available", () => {
-    store.dispatch({type: at.DISCOVERY_STREAM_LAYOUT_UPDATE, data: {layout: FAKE_LAYOUT}});
-    store.dispatch({type: at.DISCOVERY_STREAM_FEEDS_UPDATE, data: {}});
-
-    const result = selectLayoutRender(store.getState().DiscoveryStream, {}, []);
-
-    assert.lengthOf(result, 1);
-    assert.propertyVal(result[0], "width", 3);
-    assert.deepEqual(result[0].components[0], FAKE_LAYOUT[0].components[0]);
-  });
-
```

Comment:
Should we check for just store.dispatch({type: at.DISCOVERY_STREAM_FEEDS_UPDATE}); above.